sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/PrivateUtils.java

```diff
@@ -168,6 +168,19 @@ public static TPrivate createPrivate(List<TCompasTopo> compasTopos) {
         return tPrivate;
     }
 
+    /**
+     * Create private of a given type containing a simple string as value
+     * @param privateType type attribute of private element to create
+     * @param privateValue value of private to create
+     * @return the complete private
+     */
+    public static TPrivate createStringPrivate(String privateType, String privateValue) {
+        TPrivate compasLNodeStatusPrivate = new TPrivate();
```

Comment:
I suggest `tPrivate` instead of `compasLNodeStatusPrivate` for the name of the variable..